packages/pulpcore/python-galaxy-importer/python-galaxy-importer.spec

```
@@ -52,6 +52,8 @@ Requires:       python3-semantic-version >= 2.8.4
 # Remove bundled egg-info
 rm -rf %{pypi_name}.egg-info
 
+sed -i -E '/\s+ansible($|-lint)/d' setup.cfg
```

Isn't this stricter?
```suggestion
sed -i -E '/\s+ansible(-lint)?$/d' setup.cfg
```

---

Lint has a versioned dependency, and I didn't want to include the specific version.

And there are not many packages that match `^ansible-lint` anyways.

---

We could do `/\s+ansible(-lint=.*)?/d`